blocks/components/editable/index.js

```diff
@@ -1,3 +1,98 @@
-export default function Editable( { value } ) {
-	return wp.element.createElement( 'p', null, value );
+const el = wp.element.createElement;
+let editorInstance = 1;
+
+export default class Editable extends wp.element.Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.onFocusIn = this.onFocusIn.bind( this );
+		this.onFocusOut = this.onFocusOut.bind( this );
+		this.id = `tinymce-instance-${ editorInstance }`;
+		editorInstance++;
+	}
+
+	componentDidMount() {
+		this.initialize();
+		if ( this.props.focusConfig ) {
+			this.focus();
+		}
+	}
+
+	initialize() {
+		const config = {
+			mode: 'exact',
+			elements: this.id,
+			theme: false,
+			inline: true,
+			toolbar: false,
+			entity_encoding: 'raw',
+			setup: this.onSetup,
+			formats: {
+				strikethrough: { inline: 'del' }
+			}
+		};
+
+		tinymce.init( config );
+	}
+
+	onSetup( editor ) {
+		this.editor = editor;
+		editor.on( 'init', this.onInit );
+		editor.on( 'focusIn', this.onFocusIn );
+		editor.on( 'focusout', this.onFocusOut );
+	}
+
+	onInit() {
+		this.editor.setContent( this.props.value );
+	}
+
+	onChange() {
+		const value = this.editor.getContent();
+		if ( value === this.props.value ) {
+			return;
+		}
+
+		this.props.onChange( value );
+	}
+
+	onFocusIn() {
+		this.isFocused = true;
+	}
+
+	onFocusOut() {
+		this.isFocused = false;
+		this.onChange();
+	}
+
+	updateContent() {
+		let bookmark;
+		if ( this.isFocused ) {
+			bookmark = this.editor.selection.getBookmark( 2, true );
+		}
+		this.editor.setContent( this.props.value );
+		if ( this.isFocused ) {
+			this.editor.selection.moveToBookmark( bookmark );
```

Comment:
It can cause the cursor to jump to the beginning of the content editable. I think right now, it does not happen because we're not updating the content from outside the component yet.
I've had some hard time fixing these kind of bugs on the `per-block` prototype.